docs/Data Provider Nodes/google-weather.md

```diff
@@ -0,0 +1,361 @@
+---
+layout: nodes.liquid
+section: smartContract
+date: Last Modified
+title: "Google"
```

Comment:
Could we make the title more specific about what this page is?

Reply:
Wow, totally spaced on that. Thank you!

Reply:
And yes, it should be in that section!